src/Renderer/Shader/Chunk/elevation_pars_vertex.glsl

```diff
@@ -25,15 +25,13 @@
         if (mode == ELEVATION_RGBA)
             return decode32(texture2D( tex, uv ).abgr * 255.0);
         if (mode == ELEVATION_DATA || mode == ELEVATION_COLOR)
-        #if defined(WEBGL2)
             return texture2D( tex, uv ).r;
-        #else
-            return texture2D( tex, uv ).w;
-        #endif
         return 0.;
     }
 
     float getElevation(vec2 uv, sampler2D tex, vec4 offsetScale, Layer layer) {
+        // Elevation textures are inverted along the y-axis
+        uv = vec2(uv.x, 1.0 - uv.y);
```

Comment:
@jailln I know use the `uv` attribute directly added by `WebGLProgram` for a `ShaderMaterial`. Since it has the same semantics as `uv_0` (pre-PR), I decided to replace `uv_0` by `uv`. However, in `TileVS.glsl`, they were already a local variable `vec2 uv = vec2(uv_0.x, 1.0 - uv_0.y);`, which was only used as a parameter of `getElevation`.
I decided to move this code in `getElevation` as this "hack" is only needed because `x-bil` elevation texture are inverted along the y-axis.

Reply:
Ah ok thanks for the clarification :)